composer.json

```diff
@@ -157,6 +157,9 @@
             },
             "drupal/views_menu_children_filter": {
                 "Use the technique from correctActiveTrail.patch to return the original menu item": "./patches/drupal/correctMenuChildren.patch"
+            },
+            "drupal/s3fs": {
```

Comment:
Let's double-check with @ednark-gsa that these changes should go live before merging this. (Granted, it will be easy enough to reverse them.)

Reply:
Sounds good! Thanks for looking at this!